recipes/moltenvk/all/conanfile.py

```diff
@@ -23,22 +27,18 @@ class MoltenVKConan(ConanFile):
     options = {
         "shared": [True, False],
         "fPIC": [True, False],
+        "hide_vulkan_symbols": [True, False],
         "with_spirv_tools": [True, False],
         "tools": [True, False],
     }
     default_options = {
-        "shared": False,
+        "shared": True,
         "fPIC": True,
+        "hide_vulkan_symbols": False,
```

Comment:
Since this option does not actually work (yet) is it worth adding now, or just waiting for a version to get release where it does work?

Reply:
It worth because the package ID will be the same. Adding a new option later will generate another package ID.